package.json

```diff
@@ -43,7 +43,7 @@
     "percy": "percy exec -- node snapshots.js",
     "icon-svgs-to-mixins": "node scripts/convert-svgs-to-icon-mixins.js"
   },
-  "version": "2.24.1",
+  "version": "2.25.0",
```

Comment:
Do you mean to bump the version? Are you releasing right away?

Reply:
Yes I did mean to, but no I'm not releasing right away - @bartaz and I talked about this, as we're adding something we want to mention in component status, and 2.24.1 has been released, the new item should list its version as (in this case) 2.25.0, so updating the version in `package.json` made sense. 

We thought it helps keep the version number consistent across the docs, given we pull in the version from `package.json` in a few templates, like the [What's new header](https://github.com/canonical-web-and-design/vanilla-framework/blob/master/templates/docs/component-status.md#whats-new-in-vanilla-versionminor) in component status. Plus, it means there's less manual labour to do when we're ready to do a release.